install_apt_deps.sh

```diff
@@ -0,0 +1,3 @@
+sudo apt-get install python-pillow python-matplotlib python-requests python-opencv python-pyqt5
+sudo -H pip install qtawesome
+
```

Comment:
Removing this because apt doesn't have the same numbers of versions that conda supports, so we should just use conda for now and package everything nicely otherwise.